### PR TITLE
Disable account set up in production, add beta signup

### PR DESCRIPTION
**Pull request by BintLopez**
https://trello.com/c/QXxskPCv/98-disable-new-org-signup-notify-that-tool-is-in-beta

- Adds beta signup form
- Adds InterestedUser model -- each beta signup form submission is an interested user
- Disables /signup get and post in production
- Sign up button routes to beta signup

image

### CODE REVIEW

File: controllers/interestedUser.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+exports.betaSignupPost = function(req, res) {
+  interestedUser = new InterestedUser({
+    name:    req.body.name,
+    email:   req.body.email,
+    orgName: req.body.orgName
+  });
+
+  interestedUser.save(function(err) {
+    if (!err) {
+      req.flash('success', { msg: "Thanks for signing up. We'll be in touch when this VolunteerCheck is available."});
```
Comment (BintLopez) [inline controllers/interestedUser.js]: My success flash notice doesn't work and I don't know why 😿

File: controllers/interestedUser.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      res.redirect('/signup-beta')
+    } else {
+      req.flash('error', { msg: "Something went wrong. Please try again." });
```
Comment (BintLopez) [inline controllers/interestedUser.js]: Error notice works

File: controllers/interestedUser.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+var InterestedUser = require('../models/InterestedUser')
+
+/**
+ * GET /signup-beta
```
Comment (BintLopez) [inline controllers/interestedUser.js]: note -- this path cannot be /signup/beta or else it breaks img assets loading

File: controllers/interestedUser.js (hunk 1 above)
Comment (BintLopez) [inline controllers/interestedUser.js]: Note -- flashes only show up in local development if using `a.vettit:3000` vs `localhost:3000`

**Comment by BintLopez** [comment]
@lomky @WCRest another PR for reviewing 🐟 ✨

File: controllers/interestedUser.js (hunk 3 above)
Comment (lomky) [reply controllers/interestedUser.js]: huh. Any idea why?

File: views/beta-signup.handlebars [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      {{/each}}
+    </div>
+  {{/if}}
+
+    <form method="POST">
+        <div class="lcb-form">
+            <div class="input-group m-b-20">
+                <span class="input-group-addon"><i class="zmdi zmdi-account"></i></span>
+                <div class="fg-line">
+                    <input type="text" name="name" id="name" placeholder="Name" class="form-control" autofocus>
```
Comment (lomky) [inline views/beta-signup.handlebars]: Let's kill this autofocus - it scrolls us down past the explainer text

File: controllers/interestedUser.js (hunk 3 above)
Comment (BintLopez) [reply controllers/interestedUser.js]: I'm guessing the relative path to the folder changes? Does that even make sense? Lol

**Comment by BintLopez** [comment]
> I presume we're planning on pulling the InterestedUsers straight out of the DB backend?

Yep. We could also set up automated emails to alert ourselves when we have these users.

**Pull request merged**